### PR TITLE
XWIKI-15713: History tab indicate that the page does not exist when the history is empty

### PR DESCRIPTION
**Pull request by atallahade**
### Issue

https://jira.xwiki.org/browse/XWIKI-15713

### Change

* Change the warning message of an empty history of a page.

### Note

I'm making a PR because I haven't tested the change and maybe the new key should have another name.

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #set ($discard = $collectionstool.reverse($versions))
 #set ($totalVersions = $versions.size())
 #if ($totalVersions == 0)
-  #warning ($services.localization.render('thispagedoesnotexist'))
+  #warning ($services.localization.render(core.history.empty''))
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm]: You sure about that ? :)

Comment (atallahade) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm]: You are right, I should have used `core.viewers.history.*` instead ;).

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -5278,6 +5278,11 @@ xe.admin.users.filter.firstname=First name filter
 xe.admin.users.lastname=Last name
 xe.admin.users.filter.lastname=Last name filter
 
+#######################################
+## until 10.11RC1
+#######################################
+core.viewers.history.empty="The history of this page is empty."
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: `## until *`` comments as used for deprecated entries (when we stop using them)

Comment (atallahade) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: Right I should have known that. Anyway it's fixed.

**Pull request merged**